astropy/modeling/tests/test_parameters.py

```diff
@@ -294,8 +293,8 @@ def test_shift_model_parameters1d(self):
     def test_scale_model_parametersnd(self):
         sc1 = models.Scale([2, 2])
         sc1.factor = [3, 3]
-        assert np.all(sc1.factor == [3, 3])
```

Comment:
This, and all similar changes, effectively changes the meaning of the test. Is `allclose` really good enough? Is there a reason why it won't be exact match as before?

Reply:
You are right, I didn't pay much attention. I got a DeprecationWarning in one case and changed them all. I'll go back and check every test. in this case they should be exact but that's not the case in general.